enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -3867,9 +3869,9 @@ class AdjointGenerator
                                        0),
                       ConstantInt::get(Type::getInt32Ty(tapeArg->getContext()),
                                        pair.first)};
-                  op->replaceAllUsesWith(ph.CreateLoad(
+                  op->replaceAllUsesWith(ph.CreateLoad(op->getType(),
                       pair.first == -1 ? tapeArg
-                                       : ph.CreateInBoundsGEP(tapeArg, Idxs)));
+                                       : ph.CreateInBoundsGEP(cast<PointerType>(tapeArg->getType())->getElementType(), tapeArg, Idxs)));
```

Comment:
Opaque Pointers are here and we shouldn't introduce new `getElementType` (also now officially deprecated)
https://github.com/JuliaLang/julia/pull/43721/files